views/show.tmpl

```diff
@@ -1,3 +1,24 @@
+{{define "title"}}{{.PageContext.ShortName}} | {{with .PageData}}{{.Show.Title}}{{end}}{{end}}
+
+{{define "open-graph"}}
+	<meta name="twitter:card" content="summary_large_image"/>
```

Comment:
Shows might suit the [summary type](https://dev.twitter.com/cards/types/summary) rather than [large summaries](https://dev.twitter.com/cards/types/summary-large-image).